packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -407,10 +406,21 @@ export default class Meetings extends SparkPlugin {
    * Get all active meetings.
    * @returns {Object} All active meetings.
    * @public
-   * @static
    * @memberof Meetings
    */
-  static getActiveMeetings() {
-    return MeetingCollection.getAll({active: true});
+  getActiveMeetings() {
```

Comment:
We no longer need this as I don't believe active is a field we have on meeting

Reply:
@chrisadubois don't we want to be able to display the active meetings though? When/why was the field removed?